docs/fundamentals/accessibility.md

```diff
@@ -0,0 +1,378 @@
+                   SemanticProperties.Description="Progress indicator."
+                   SemanticProperties.Hint="Saving your data to the cloud." />
+```
+
+Alternatively, it can be set in C#:
+
+```csharp
+ActivityIndicator activityIndicator = new ActivityIndicator
+{
+    IsRunning = true
+};
+SemanticProperties.SetDescription(activityIndicator, "Progress indicator.");
+SemanticProperties.SetHint(activityIndicator, "Saving your data to the cloud.");
+```
+
+In addition, the `SetValue` method can also be used to set the `SemanticProperties.Hint` attached property:
+
+```csharp
+activityIndicator.SetValue(SemanticProperties.HintProperty, "Saving your data to the cloud.");
+```
+
+On Android, this property behaves slightly differently depending on the control it's attached to. For example, for controls without text values, such as `Switch` and `CheckBox`, the controls will display the hint with the control. However, for controls with text values, the hint is not displayed and is read after the text value.
+
+> [!WARNING]
+> The `SemanticProperties.Hint` property conflicts with the `Entry.Placeholder` property on Android, which both map to the same native property. Therefore, setting a different `SemanticProperties.Hint` value to the `Entry.Placeholder` value isn't recommended.
+
+<!-- ^ and Editor? -->
+
+### Heading levels
+
+The `SemanticProperties.HeadingLevel` attached property enables an element to be marked as a heading to organize the UI and make it easier to navigate. Some screen readers enable users to quickly jump between headings.
+
+Headings have a level from 1 to 9, and are represented by the `SemanticHeadingLevel` enumeration, which defines `None`, and `Level1` through to `Level9` members.
+
+> [!IMPORTANT]
+> While Windows offers 9 levels of headings, Android and iOS only offer a single heading. Therefore, when `SemanticProperties.HeadingLevel` is set on Windows it maps to the correct heading level. However, when set on Android and iOS it maps to a single heading level.
+
+The following example demonstrates setting this attached property:
+
+```xaml
+<StackLayout SemanticProperties.Description="Main page layout"
+             SemanticProperties.HeadingLevel="Level1">
+    ...
+</StackLayout>
+```
+
+Alternatively, it can be set in C#:
+
+```csharp
+StackLayout stackLayout = new StackLayout();
+SemanticProperties.SetDescription(stackLayout, "Main page layout.");
+SemanticProperties.SetHeadingLevel(stackLayout, SemanticHeadingLevel.Level1);
+```
+
+In addition, the `SetValue` method can also be used to set the `SemanticProperties.HeadingLevel` attached property:
+
+```csharp
+stackLayout.SetValue(SemanticProperties.HeadingLevelProperty, SemanticHeadingLevel.Level1);
+```
+
+<!-- Todo: is this right?
+### WebView    
+
+The native WebView control on each platform is responsible for determining which elements of its content are accessible, and each WebView control should report its HTML content as accessibility elements.
```

Comment:
what do you mean by "each WebView control should report its HTML content as accessibility elements."?

with WebView controls, I think it's actually on the specific website / HTML content. If someone is using a WebView with a website that's not accessible, then it won't be accessible through us either, whereas if it is, then it will be.

@PureWeen do you have additional thoughts?